dask_glm/regularizers.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+
+class L2(object):
```

Comment:
In general, i am not a big fan of classes with just static methods. If name spacing is important, we should pull them out into separate files/modules. Alternatively, we could use a factory class that configures these methods.